Makefile

```diff
@@ -25,6 +25,10 @@ ifndef UNAME_M
 UNAME_M := $(shell uname -m)
 endif
 
+ifndef ENABLE_WERROR
+	ENABLE_WERROR ?= 0
+endif
+
```

Comment:
I don't think these lines are necessary - if it's blank it won't equal 1 either.

Reply:
@cebtenzzre  Yup, will remove it :)

Thank you for taking the time to review the PR! 😄